### PR TITLE
Add pages to install and manage lxd

### PR DESCRIPTION
**Pull request by edlerd**
## Done

- added lxd/install and lxd/manage pages

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- check /lxd, /lxd/install and /lxd/manage

## Issue / Card

Fixes WD-4823

### CODE REVIEW

File: navigation.yaml [hunk 1]
```diff
@@ -415,6 +415,14 @@ lxd:
   children:
     - title: Overview
       path: /lxd
+    - title: Install
+      path: /lxd/install
+    - title: Manage
+      path: /lxd/manage
+    - title: Docs
+      path: https://documentation.ubuntu.com/lxd/
```
Comment (anthonydillon) [inline navigation.yaml:423]: This is still private it looks like

File: templates/appliance/lxd/_next-steps.html [hunk 2]
```diff
@@ -32,7 +32,7 @@ <h2>
         <pre>lxc remote add appliance IP-ADDRESS
 lxc list appliance:</pre>
         <p>
-          And you're away. For more information check out the <a href="https://linuxcontainers.org">LXD website</a> or read our <a href="https://discuss.linuxcontainers.org/t/lxd-cluster-on-raspberry-pi-4/9076">tutorial on turning the LXD Ubuntu Appliance into your own homelab</a>.
+          And you're away. For more information check out the <a href="https://ubuntu.com/lxd">LXD website</a> or read our <a href="https://discuss.linuxcontainers.org/t/lxd-cluster-on-raspberry-pi-4/9076">tutorial on turning the LXD Ubuntu Appliance into your own homelab</a>.
```
Comment (anthonydillon) [inline templates/appliance/lxd/_next-steps.html:35]:
```suggestion
          And you're away. For more information check out the <a href="/lxd">LXD website</a> or read our <a href="https://discuss.linuxcontainers.org/t/lxd-cluster-on-raspberry-pi-4/9076">tutorial on turning the LXD Ubuntu Appliance into your own homelab</a>.
```

File: templates/kubernetes/charmed-k8s.html [hunk 3]
```diff
@@ -238,7 +238,7 @@ <h3 class="p-heading--4">Containers for legacy apps</h3>
       <ul class="p-list">
         <li class="p-list__item is-ticked">Use the Linux Containers Daemon, LXD, for pure-container guests in OpenStack.</li>
         <li class="p-list__item is-ticked">Faster to launch, with zero latency and amazing quality of service, LXD guests are the bridge between traditional VMs and next-generation Kubernetes.</li>
-        <p><a href="https://linuxcontainers.org/">Visit the LXD website&nbsp;&rsaquo;</a></p>
+        <p><a href="https://ubuntu.com/lxd">Visit the LXD website&nbsp;&rsaquo;</a></p>
```
Comment (anthonydillon) [inline templates/kubernetes/charmed-k8s.html:241]:
```suggestion
        <p><a href="/lxd">Visit the LXD website&nbsp;&rsaquo;</a></p>
```

File: templates/kubernetes/index.html [hunk 4]
```diff
@@ -268,7 +268,7 @@ <h3 class="p-card__title">Canonical Kubernetes benefits</h3>
       <h2 class="p-logo-section__title u-align--left">Fully-support Kubernetes clouds</h2>
       <div class="p-logo-section__items">
         <div class="p-logo-section__item">
-          <a href="https://linuxcontainers.org/">
+          <a href="https://ubuntu.com/lxd">
```
Comment (anthonydillon) [inline templates/kubernetes/index.html:271]:
```suggestion
          <a href="/lxd">
```

File: templates/templates/navigation-developer-h.html [hunk 5]
```diff
@@ -208,7 +208,7 @@ <h4 class="p-muted-heading--small">Tools</h4>
             <li class="p-inline-list__item"><a href="https://netplan.io">netplan</a></li>
             <li class="p-inline-list__item"><a href="https://juju.is">Juju</a></li>
             <li class="p-inline-list__item"><a href="https://snapcraft.io">Snapcraft</a></li>
-            <li class="p-inline-list__item"><a href="http://linuxcontainers.org">LXD</a></li>
+            <li class="p-inline-list__item"><a href="https://ubuntu.com/lxd">LXD</a></li>
```
Comment (anthonydillon) [inline templates/templates/navigation-developer-h.html:211]:
```suggestion
            <li class="p-inline-list__item"><a href="/lxd">LXD</a></li>
```

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Few absolute paths that should be relative.

**Review by anthonydillon — COMMENTED** [review]
Some QA issues around the HTML validation of /lxd/install.

`<pre>` inside a `<p>` break out of the p and confuses the markup.

File: templates/lxd/install.html [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+{% block content %}
+
+<section class="p-strip--suru-topped">
+  <div class="row u-equal-height u-vertically-center">
+    <div class="col-6">
+      <h1>Install LXD</h1>
+      <p>Try LXD on your laptop, workstation or server. Set up a single instance LXD for testing and development, or run it in a clustering mode for highly-available production environments.</p>
+      <p>Looking for help running LXD?</p>
+      <p>
+        <a class="p-button--positive js-invoke-modal" href="/lxd#get-in-touch">Get in touch</a>
```
[the remark this answers is not in the document]
Comment (edlerd) [reply templates/lxd/install.html]: replaced with a link to the static contact us form.

File: navigation.yaml (hunk 1 above)
Comment (edlerd) [reply navigation.yaml]: replaced with the correct docs link

File: templates/lxd/manage.html [hunk 7]
```diff
@@ -0,0 +1,310 @@
...
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-bordered">
+  <div class="row u-sv3">
+    <div class="col-6">
+      <h2>LXD graphical user interface</h2>
+      <p>An official LXD UI tool is now available as an experimental feature. The UI supports most of the functionalities surrounding managing instances. More features coming soon.</p>
+      <p><a href="https://ubuntu.com/blog/lxd_ui">Read more about the current state of the UI and future plans</a></p>
```
Comment (anthonydillon) [inline templates/lxd/manage.html:51]:
```suggestion
      <p><a href="/blog/lxd_ui">Read more about the current state of the UI and future plans</a></p>
```

File: templates/lxd/manage.html [hunk 8]
```diff
@@ -0,0 +1,310 @@
...
+    </div>
+    <div class="col-6 u-vertically-center u-align--center u-hide--small u-hide--medium">
+      {{ image (
+        url="https://lh4.googleusercontent.com/oURP8ffnOR44PGUCGaTlD1rRtfVN3oBkPxuLpuvvI4vrYt_093UhOiDTwIihUtlfPWTEA_48rMqZL1wB832Pa4I5AppqTl4EHbcTPRf3V-fxFQaae2jdYcA7-ntt-vOl6wgTYl1hi54RTifHPnOgnIY",
```
Comment (anthonydillon) [inline templates/lxd/manage.html:55]: Can you put this image on assets please?

**Review by anthonydillon — APPROVED** [review]
LGTM with a very small suggestion in the code. Sorry I missed it before.

File: templates/lxd/manage.html [hunk 9]
```diff
@@ -0,0 +1,310 @@
...
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-bordered">
+  <div class="row u-sv3">
+    <div class="col-6">
+      <h2>LXD graphical user interface</h2>
+      <p>An official LXD UI tool is now available as an experimental feature. The UI supports most of the functionalities surrounding managing instances. More features coming soon.</p>
+      <p><a href="/blog/lxd_ui">Read more about the current state of the UI and future plans</a></p>
```
Comment (anthonydillon) [inline templates/lxd/manage.html:51]: Underscore? 🥴

**Review by anthonydillon — APPROVED** [review]
LGTM v👍

**Pull request merged**